app/config/variables.php

```diff
@@ -748,7 +748,7 @@
                 'name' => '_APP_EXECUTOR_HOST',
                 'description' => 'The host used by Appwrite to communicate with the function executor!',
                 'introduction' => '0.13.0',
-                'default' => 'http://appwrite-executor/v1',
+                'default' => 'http://executor/v1',
```

Comment:
Looks good, but maybe we should go back to using `appwrite-executor`, or have smart migration script (if value is exactly `http://appwrite-executor/v1`, then replace. If not, keep using their modified value)

Reply:
let's leave it as executor. smarter installation command coming in: https://github.com/appwrite/appwrite/issues/6137

Reply:
@stnguyen90 the override PR has been merged. I think now we should change it back to `appwrite-executor`